### ISSUE TITLE
Require a file or variants found for asset: packages/alarm/assets/not_blank.mp3.

### ISSUE THREAD

**Issue by KetutKusuma** [gdelataillade/alarm#115]
**Alarm plugin version**
*ex: 3.0.0-dev.4*

**Describe the bug**
*after finish add the package when i run again my app, that can not run because this error "No file or variants found for asset: packages/alarm/assets/not_blank.mp3", i don't know if that is my fault the error is show up, please fix bug or if it is not a bug please tell me how to fix it sir*

**To Reproduce**
*Steps to reproduce the behavior:*
1. *...*

**Expected behavior**
*A clear and concise description of what you expected to happen.*

**Screenshots**

<img width="908" alt="image" src="https://github.com/gdelataillade/alarm/assets/71806737/1b045e41-61fe-4784-b0b3-976170a80750">

**Device info**
*Provide device info (Manufacturer, OS version, ...)*
*Ex: iPhone 13 Pro, iOS 16.2, ...*

**Additional context**
*Add any other context about the problem here.*

**Comment by gdelataillade** [comment]
Hi @KetutKusuma 

My bad, it looks like I made a mistake in the latest dev release. I'll publish version `3.0.0-dev.5` with a fix today or tomorrow. I'll keep you updated. If you want it to work now, I'm not sure but you can try to remove the alarm's `.pubignore` file.

**Comment by KetutKusuma** [comment]
Yeah it works very well, you did great job bro. Now i can use this package to help me, thank you bro👍

**Comment by gdelataillade** [comment]
Awesome !
